apis/v1beta1/types.go

```diff
@@ -280,7 +280,8 @@ type NetworkDeviceSpec struct {
 	Gateway6 string `json:"gateway6,omitempty"`
 
 	// IPAddrs is a list of one or more IPv4 and/or IPv6 addresses to assign
-	// to this device.
+	// to this device.  IP addresses must also specify the segment length in 
```

Comment:
```suggestion
	// to this device.  IP addresses must also specify the segment length in
```
@rvanderp3 this change in all 3 files should make the linter happy

Reply:
Maybe also rebase onto main